Samples/Samples/Converters/EndTypeToBoolConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Samples.ViewModel;
+using Xamarin.Forms;
+
+namespace Samples.Converters
+{
+    public class EndTypeToBoolConverter : IValueConverter
```

Comment:
Change class name to RecurrenceEndTypeToBoolConverter

Reply:
Have altered the class as suggested, but what would you suggest for the exposing a different property?